AzureMonitorAgent/agent.py

```diff
@@ -532,8 +532,11 @@ def enable():
         log_and_exit("Enable", GenericErrorCode, "Failed to add environment variables to {0}: {1}".format(config_file, e))
 
     if "ENABLE_MCS" in default_configs and default_configs["ENABLE_MCS"] == "true":
-        start_amacoreagent()
-        restart_launcher()
+        if platform.machine() != 'aarch64':
+            # enable processes for Custom Logs
+            ensure["azuremonitor-agentlauncher"] = True
+            ensure["azuremonitor-coreagent"] = True
+            
```

Comment:
there's also line 545 (github not letting me comment there ...) another instance of `start_amacoreagent`, that should probably be changed as well to ensure

Reply:
linter we have in packaging script would have found it 👍